### ISSUE TITLE
Add changeset tag for address additions when none of the address points have been moved

### ISSUE THREAD

**Issue by watmildon** [facebook/Rapid#965]
### Description

It is very common for address data to come from GIS departments with nodes at the parcel centroid. It is common OSM practice (but not always the case!) to move those nodes to buildings before addition. Unfortunately, it is also somewhat common to have mappers skip this relocation step. It would be nice to be able to find such changeset easily reach out to mappers making this common error and let them know how they can improve their mapping.

An example changeset that could be improved with a bit of node relocation: https://www.openstreetmap.org/changeset/135995246

An example changeset where the nodes have been aligned with buildings in imagery: https://www.openstreetmap.org/changeset/137627739

There may be other things Rapid can do to help here, warnings, information prompts etc but I'm not super familiar with the kinds of things you've found effective. Happy to hear any ideas.

**Comment by Bonkles** [comment]
This is a difficult area to talk about, but I appreciate the spirit in which you are requesting this sort of thing. None of us want to see a tool misused, and we want to give folks more of a feeling that the human in this edit loop is of paramount importance, not simply a mechanical turk that should click as fast as they can. 

I can see some value in the idea of tagging changes that were not modified at all, but I worry that this might have a greater net negative effect.  My reasoning: 

1) Honest folks might feel uneasy that we're beginning to ascribe behavioral patterns to their edits. 
2) The 'edit-at-warp-speed' folks would just learn that they need to move a single node two pixels to get around the message/tag.

Then all we've succeeded in doing is scaring the honest folks out of editing, and we've barely inconvenienced the folks we're concerned about.

**Comment by watmildon** [comment]
I'm definitely sympathetic to not wanting to harass well intentioned users! I wouldn't even say Rapid is being "misused" so much as "not used to it's full potential". I generally assume that it's a lack of common understanding and not anything with malicious intent. My only goal is to help well intentioned mappers do their best mapping. I don't dare try to come up with ways to combat intentional avoidance. As you've noted, folks will skirt barriers if they wish.

With that in mind, there's already a large suite of tools that try to detect uncommon behavior and give nudges to users to help keep everyone rowing in the same direction: OSM Inspector, iD and JOSM validators, etc etc. It's not clear to me that this check is any more behavior ascribing than we already have in the overall ecosystem. There's nothing necessarily stopping OSM Inspector or similar from doing this same detection. Because Rapid already has the data at the time of addition, it's best positioned to do whatever prompting/tagging/evaluation.

I say this as someone who has likely reviewed+added more of this address data into OSM than anyone, adding 1000+ addresses in a changeset without moving a node is always caused by mapping that could be crisper.

Anyhow, this class of error typically doesn't seriously damage the map so there's no particular rush to get something rolled out. If there's some light touch solution that's probably best.

**Comment by Bonkles** [comment]
Got it. Perhaps there's room for another changeset tag for this sort of thing. We can *certainly* be better about detecting actual point conflation problems, such as detecting if a newly added address point shares data with a building it is on top of, that sort of thing would lead to better edits esp. from new mappers.

**Comment by watmildon** [comment]
Yes! Would love to brainstorm various checks that can help keep folks on a good path. I'll have to spend some time in Rapid (I'm usually in JOSM using MWAI plugin) to get a sense of other things that could be helpful. I'll add it to my todo list.
